src/wasm-lib/kcl/src/executor.rs

```diff
@@ -191,7 +188,7 @@ impl EnvironmentRef {
 
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, ts_rs::TS, JsonSchema)]
 pub struct Environment {
-    bindings: HashMap<String, KclValue>,
+    bindings: IndexMap<String, KclValue>,
```

Comment:
Any reason to prefer IndexMap over BTreeMap?

Reply:
BTreeMap keeps things in sorted order according to the key type's `Ord` trait. IndexMap preserves insertion order. Unless we want to sort things, insertion order is a better, more intuitive default, in my opinion. For bindings specifically, it will keep variables in the order that they were defined in the source. That's pretty nice.